hack/presubmit.sh

```diff
@@ -84,8 +84,7 @@ update_crdgen() {
   which controller-gen || go install github.com/TheSpiritXIII/controller-tools/cmd/controller-gen@v0.14.1-gmp
```

Comment:
Thought: Irrelevant to this PR, butt this should probably be moved to Bingo.

Reply:
👍🏽  let's do in the next one